core/server/api_container/server/startosis_engine/interpretation_time_value_store/service_value_repository_test.go

```diff
@@ -0,0 +1,93 @@
+package interpretation_time_value_store
+
+import (
+	port_spec_core "github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/port_spec"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/service"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/database_accessors/enclave_db"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/kurtosis_types"
+	"github.com/kurtosis-tech/kurtosis/core/server/api_container/server/startosis_engine/kurtosis_types/port_spec"
+	"github.com/stretchr/testify/require"
+	bolt "go.etcd.io/bbolt"
+	"go.starlark.net/starlark"
+	"go.starlark.net/starlarkstruct"
+	"os"
+	"testing"
+)
+
+const (
+	starlarkThreadName     = "thread-for-db-test"
+	serviceName            = service.ServiceName("datastore-1")
+	serviceNameStarlarkStr = starlark.String(serviceName)
+	hostName               = serviceNameStarlarkStr
+	ipAddress              = starlark.String("172.23.34.44")
+)
+
+func TestAddAndGetTest(t *testing.T) {
```

Comment:
maybe add a test for the service doesn't exist case and the case where the service was stored as an empty value to illustrate/test those behaviors?